movie.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+require_relative 'source'
+
+class Movie < Item
+  attr_accessor :silent
+
+  def initialize(publish_date, silent, archived, name)
+    super(publish_date, archived, name)
+    @silent = silent
+  end
```

Comment:
Remember that the last `Item` class parameter is _optional_, so you must also make it optional when creating the `Movie` class, otherwise, you would need to manually write the id.
Also, the last parameter is called `id` not `name`.

Reply:
@Yothu  Do you mean I can leave out the ID?

Reply:
I mean that you need to leave it the same as in the `Item` class, with a default, you can use a function that generates random numbers.

Reply:
Okay cool. I have decided not to add it for now. It will still work. Please review and approve it